src/ClusterBootstrap/bash_step_by_step_deploy.sh

```diff
@@ -0,0 +1,26 @@
+./deploy.py -y build
```

Comment:
This can be replaced by script block?

Reply:
sure. but later 2 commands are executed with az_tools.py. do you want to merge them into the script block by running bash cmd in deploy.py? like some "run_command_on_devbox"